CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - FORECAST RECAP LINE : Add reference to display related invoice, expense, sale order, purchase order or forecast reason
 - MrpForecast : Enable Tracking Feature for all fields on the creation of record
 - SALE/PURCHASE ORDER : Addtion of boolean field 'isAnalyticDistributionRequired' in app's configuration.
+- HR : add traking on 'status' of timesheet-form.
```

Comment:
its 'tracking' not 'traking'
change changelog message to 
HR : 'Enable Tracking Feature for 'statusSelect' field.